app/graphql/types/well_mutation_type.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Types
+  # The type for Well mutations.
+  class WellMutationType < BaseObject
```

Comment:
Similary does it make sense to add this to a Types::Migrations namespace structure?

Reply:
Resolved as per comment on similar conversation